jsm.go

```diff
@@ -100,15 +100,18 @@ type StreamConfig struct {
 	DenyDelete        bool            `json:"deny_delete,omitempty"`
 	DenyPurge         bool            `json:"deny_purge,omitempty"`
 	AllowRollup       bool            `json:"allow_rollup_hdrs,omitempty"`
+	AllowDirect       bool            `json:"allow_direct,omitempty"`
 
 	// Allow republish of the message after being sequenced and stored.
-	RePublish *SubjectMapping `json:"republish,omitempty"`
+	RePublish *RePublish `json:"republish,omitempty"`
```

Comment:
Do we want to introduce breaking change here for the sake of changing the struct name?

Reply:
Was not released, was experimental so yes we can do that at this point I believe.

Reply:
That's fair,  thanks.